README.md

```diff
@@ -9,6 +9,22 @@ a unique identifier and transformation get one as well, this will allow for crea
 that are not explicitly defined, but instead derived from following matching representation and
 transformation pairs.
 
+## Quick-Start
+
+The [`example_files`](/example_files/) folder contains an English to French conjucation file. Generate the learnables by running
+
+```
+cargo run --example generate_from_text -- example_files/englins_to_french_verbs.txt --output example_files/eng_fr_learnables.yaml
```

Comment:
`englins` -> `english`

I actually wrote the `generate_from_text` helper to be able to easily load a file that had the following content;
```
0|0
10|16
20|32
30|48
40|64
50|80
60|96
70|112
80|128
90|144
a0|160
b0|176
c0|192
d0|208
e0|224
f0|240
```
but I'm happy to take the example of the French / English words. We could also add this one as well, your call.

Reply:
I've added the a `hex_to_dec_conversions.txt` file to incorporate what you have here. Simplified the spelling of `english` to prevent further confusion.